src/main/java/com/epam/training/auction_backend/services/AuctionService.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.training.auction_backend.services;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.epam.training.auction.common.AuctionTransferObject;
+import com.epam.training.auction.common.AuctionsService;
+import com.epam.training.auction_backend.entity.Auction;
+import com.epam.training.auction_backend.entity.User;
+
+public final class AuctionService implements AuctionsService {
+	
+	private static final Logger logger = Logger.getLogger(AuctionService.class);
+	
+	@Autowired
+	private AuctionRepository repository;
```

Comment:
It's better to do autowiring in this case through constructor DI as repository is mandatory object in this case